scripts/listen_to_events/strategies.py

```diff
@@ -275,11 +299,19 @@ async def format_data(self, chain, event):
 
     def discord_channels(self):
         return os.getenv("BALANCER_NEW_POOL_DISCORD_CHANNEL_IDS", "").split(",")
+    
+    async def get_pool_address(self, event):
+        data = parse_event_topics(event)
+        return "0x" + data["poolAddress"][-40:]
```

Comment:
👀 why the cutoff at -40?

Reply:
Im not sure, I got this from other function on this same `Strategy`